tools/osm_mine_downloader.py

```diff
@@ -0,0 +1,77 @@
+import osm2geojson
+import geopandas as gpd
+import requests
+import json
+
+# Bounding boxes
+lat_bounds = [-90 + x for x in range(0,180)]
+long_bounds = [-180 + 5*x for x in range(0,360)]
```

Comment:
why `+5*x`? should it instead be `long_bounds=range(-180,180,5)`?

Reply:
Yep that's a cleaner way to do it! Increments of 5 are done to reduce the number of queries made to the Overpass server, whilst also not having each query be so large that it times out.